setup.cfg

```diff
@@ -33,12 +33,14 @@ test =
     graphviz
     coverage
     spectral-cube
+    stdatamodels>=1.1.0
 docs =
-    # TODO: Remove numpy pinning when astropy v5.2.1 is released.
-    numpy<1.24
     sphinx-astropy
     matplotlib
     graphviz
+    # TODO: Remove pinning when ndcube fixed
+    # See https://github.com/astropy/specutils/issues/1028
+    ndcube<2.1
```

Comment:
NDCube 2.1.1 was released for the fix with this, we should be able to remove this pin now.

Reply:
No, there is still https://github.com/sunpy/ndcube/issues/604

Reply:
Ah, blast. Alright, it stays for now 😆